contracts/ERC1238/ERC1238.sol

```diff
@@ -85,6 +83,26 @@ contract ERC1238 is IERC1238 {
         return batchBalances;
```

Comment:
We should probably delineate the naming of these functions to match our usage of `batch` and `bundle` where `batch` is multiple tokens for a single address and `bundle` is multiple addresses.

In this case this function should be `function balanceOfBatch(address account, uint256[] memory ids)`

and we have another that is `function balanceOfBundle(address[] memory accounts, uint256[][] memory ids)`

Reply:
Great suggestion, I added it